### PR TITLE
CB-5153 Change description for 'Save credentials' mark on connection auth pop-up + unify

### PR DESCRIPTION
**Pull request by sergeyteleshev**

### CODE REVIEW

File: webapp/packages/plugin-connections/src/ConnectionAuthService.ts [hunk 1]
```diff
@@ -40,6 +44,20 @@ export class ConnectionAuthService extends Dependency {
...
+    }
+
+    const project = this.projectInfoResource.get(projectId);
+
+    if (!project) {
+      return false;
+    }
+
+    return isSharedProject(project);
+  }
```
Comment (Wroud) [inline webapp/packages/plugin-connections/src/ConnectionAuthService.ts]: really strange public method in `ConnectionAuthService` that accepts `projectId`
`connectionAuthService.isConnectionShared(projectId)`
why auth service know shared connection or not?
what connection is shared?
how it depends on projectId?

File: webapp/packages/plugin-connections/src/ConnectionAuthentication/ConnectionAuthenticationDialog.tsx [hunk 2]
```diff
@@ -57,6 +58,7 @@ export const ConnectionAuthenticationDialog: DialogComponent<ConnectionAuthentic
               formId={payload.config.connectionId || payload.driverId}
               className={s(styles, { connectionAuthenticationFormLoader: true })}
               hideFeatures={['nonSecuredProperty']}
+              distributed={payload.distributed}
```
Comment (Wroud) [inline webapp/packages/plugin-connections/src/ConnectionAuthentication/ConnectionAuthenticationDialog.tsx]: seems like wrong dependency propagation, why not to use `useResource` at place where we need it?

File: webapp/packages/plugin-settings-administration/src/locales/ru.ts [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
 export default [
   ['plugin_settings_administration_settings_save_success', 'Настройки сохранены успешно'],
   ['plugin_settings_administration_settings_save_fail', 'Ошибка сохранения настроек'],
-  ['plugin_settings_administration_drawer_item_title', 'Настройки'],
+  ['plugin_settings_administration_drawer_item_title', ' Глобальные Настройки'],
```
Comment (EvgeniaBzzz) [inline webapp/packages/plugin-settings-administration/src/locales/ru.ts]: 'Глобальные настройки'
Second word in lowercase

File: webapp/packages/plugin-connections/src/ConnectionAuthService.ts [hunk 4]
```diff
@@ -86,6 +104,7 @@ export class ConnectionAuthService extends Dependency {
         connection: key,
         networkHandlers,
         resetCredentials,
+        distributed: this.serverConfigResource.distributed,
```
Comment (Wroud) [inline webapp/packages/plugin-connections/src/ConnectionAuthService.ts]: seems like it's better to get this data in dialog instead (cuz it's not a parameter but app env)

File: webapp/packages/plugin-connections/src/ConnectionAuthentication/NetworkHandlerAuthForm.tsx [hunk 5]
```diff
@@ -70,7 +77,16 @@ export const NetworkHandlerAuthForm = observer<Props>(function NetworkHandlerAut
...
+          label={translate(
+            distributed
+              ? 'connections_connection_authentication_save_credentials_for_user'
+              : 'connections_connection_authentication_save_credentials_for_session',
+          )}
+          title={translate(
+            distributed
+              ? 'connections_connection_authentication_save_credentials_for_user_tooltip'
+              : 'connections_connection_authentication_save_credentials_for_session_tooltip',
+          )}
```
Comment (Wroud) [inline webapp/packages/plugin-connections/src/ConnectionAuthentication/NetworkHandlerAuthForm.tsx]: not sure that this setting reflects on SSH/SSL

File: webapp/packages/plugin-connections/src/ConnectionForm/ConnectionFormService.ts [hunk 6]
```diff
@@ -153,6 +155,7 @@ export class ConnectionFormService {
       authModelId: credentialsState.authModelId,
       networkHandlers: credentialsState.networkHandlers,
       driverId: config.driverId,
+      distributed: this.serverConfigResource.distributed,
```
Comment (Wroud) [inline webapp/packages/plugin-connections/src/ConnectionForm/ConnectionFormService.ts]: same here (it's not a param)

File: webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts [hunk 7]
```diff
@@ -80,20 +80,6 @@ export class ConnectionOptionsTabService extends Bootstrap {
...
-
-    const project = this.projectInfoResource.get(state.projectId);
-
-    if (!project) {
-      return false;
-    }
-
-    return isSharedProject(project);
-  }
-
```
Comment (Wroud) [inline webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts:100]: seems like it should be reverted

**Review by Wroud — CHANGES_REQUESTED** [review]

File: webapp/packages/plugin-connections/src/ConnectionAuthentication/NetworkHandlerAuthForm.tsx (hunk 5 above)
Comment (sergeyteleshev) [reply webapp/packages/plugin-connections/src/ConnectionAuthentication/NetworkHandlerAuthForm.tsx]: it is. all the cases we found and clarified with QA are in the task description

File: webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts (hunk 7 above)
Comment (sergeyteleshev) [reply webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts:100]: moved it to the `projectInfoResource` cause it is needed everywhere

**Review by Wroud — APPROVED** [review]

**Pull request merged**